### PR TITLE
added breadcrumbs

### PR DESCRIPTION
**Pull request by nehajain008**
Story AB#15190
Task AB#15651

### CODE REVIEW

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -10,152 +10,154 @@
...
-                <li class="nhsuk-header__navigation-item">
-                    <a class="nhsuk-header__navigation-link" href="#">
-                        Create or manage orders
-                    </a>
+                        <span class="nhsuk-u-visually-hidden">Close menu</span>
+                    </button>
+                </p>
+                <ul class="nhsuk-header__navigation-list">
+                    <li class="nhsuk-header__navigation-item nhsuk-header__navigation-item--for-mobile">
+                        <a class="nhsuk-header__navigation-link" href="/">
```
Comment (leerogers-bjss) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Shared/_Layout.cshtml:30]: this needs to be an asp-action / asp-controller tag, not a href

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Shared/_Layout.cshtml [hunk 2]
```diff
@@ -10,152 +10,154 @@
...
+                            Home
+                            <svg class="nhsuk-icon nhsuk-icon__chevron-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
+                                <path d="M15.5 12a1 1 0 0 1-.29.71l-5 5a1 1 0 0 1-1.42-1.42l4.3-4.29-4.3-4.29a1 1 0 0 1 1.42-1.42l5 5a1 1 0 0 1 .29.71z"></path>
+                            </svg>
+                        </a>
+                    </li>
+                    <li class="nhsuk-header__navigation-item">
+                        <a class="nhsuk-header__navigation-link" href="/">
```
Comment (leerogers-bjss) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Shared/_Layout.cshtml:38]: same as above, needs to be an asp-action / asp-controller, not href

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Shared/_Layout.cshtml [hunk 3]
```diff
@@ -10,152 +10,154 @@
...
+                            Catalogue Solutions
+                        </a>
+                    </li>
+                    <li class="nhsuk-header__navigation-item">
+                        <a class="nhsuk-header__navigation-link" href="#">
```
Comment (leerogers-bjss) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Shared/_Layout.cshtml:43]: same as above, needs to be an asp-action / asp-controller, not href

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Shared/_Layout.cshtml [hunk 4]
```diff
@@ -10,152 +10,154 @@
...
+                            Create or manage orders
+                        </a>
+                    </li>
+                    <li class="nhsuk-header__navigation-item">
+                        <a class="nhsuk-header__navigation-link" href="#">
```
Comment (leerogers-bjss) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Shared/_Layout.cshtml:48]: same as above, needs to be an asp-action / asp-controller, not href

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Shared/_Layout.cshtml [hunk 5]
```diff
@@ -10,152 +10,154 @@
...
+                    </li>
+                </ul>
+            </div>
+        </nav>
+    </header>
+    <nav class="nhsuk-breadcrumb" aria-label="Breadcrumb">
+        <div class="nhsuk-width-container">
+            <ol class="nhsuk-breadcrumb__list">
+                <li class="nhsuk-breadcrumb__item">
+                    <a class="nhsuk-breadcrumb__link" href="/">Home</a>
```
Comment (leerogers-bjss) [inline src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Shared/_Layout.cshtml:60]: same as above, needs to be an asp-action / asp-controller, not href

**Review by leerogers-bjss — CHANGES_REQUESTED** [review]

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Shared/_Layout.cshtml (hunk 1 above)
Comment (nehajain008) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Shared/_Layout.cshtml]: Done.

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Shared/_Layout.cshtml (hunk 2 above)
Comment (nehajain008) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Shared/_Layout.cshtml]: Done.

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Shared/_Layout.cshtml (hunk 3 above)
Comment (nehajain008) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Shared/_Layout.cshtml]: Done.

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Shared/_Layout.cshtml (hunk 4 above)
Comment (nehajain008) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Shared/_Layout.cshtml]: Done.

File: src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Shared/_Layout.cshtml (hunk 5 above)
Comment (nehajain008) [reply src/NHSD.GPIT.BuyingCatalogue.WebApp/Areas/Solutions/Views/Shared/_Layout.cshtml]: Done.

**Review by leerogers-bjss — APPROVED** [review]

**Pull request merged**